fast64_internal/oot/c_writer/oot_level_c.py

```diff
@@ -839,3 +872,21 @@ def __init__(self):
         self.roomMainC = {}
         self.roomMeshInfoC = {}
         self.roomMeshC = {}
+
+
+def ootGetHeaderDefines(room, headerIndex):
+    """Returns CData containing defines for actor and object lists lengths"""
+    data = CData()
+
+    if len(room.objectList) > 0:
+        data.header += (
+            "#define LENGTH_" + str(room.objectListName(headerIndex)).upper() + " " + str(len(room.objectList))
```

Comment:
I think instead of having the `LENGTH_` name generated in two places (and risking them to go out of sync) you could add a function to `OOTRoom` similar to `objectListName` and do like
```suggestion
            "#define " + str(room.objectListLengthDefineName(headerIndex)) + " " + str(len(room.objectList))
```
(`objectListLengthDefineName(` is only a suggestion :monkasteer:)